CHANGELOG.md

```diff
@@ -1,12 +0,0 @@
-## [0.9.1] - 23-06-2016
-### Added 
-- Added files property in package.json
-- Added first simple test
-- Added travis config
-
-### Change
-- Update .gitignore
-- Update path in package.json to posthtml
-- Change #2, #3, Downgrade to ES5
-- Update devDep
-- Update repo path in readme
```

Comment:
Just a note, I copied this into github's releases [here](https://github.com/posthtml/posthtml-loader/releases/tag/v0.9.1)